plugins/net_plugin/net_plugin.cpp

```diff
@@ -1558,9 +1558,6 @@ namespace eosio {
       if( !ec ) {
          my_impl->dispatcher->retry_fetch( shared_from_this() );
       } else if( ec == boost::asio::error::operation_aborted ) {
```

Comment:
actually now you don't need the `else if`

Reply:
The else-if is to avoid the `peer_elog` of the `else`